1/contrib/jenkins/run-jenkins

```diff
@@ -6,24 +6,17 @@
 #
 # It also sets the admin password to ${JENKINS_PASSWORD}.
 #
+source /usr/local/bin/jenkins-common.sh
 
-# Generate passwd file based on current uid
-function generate_passwd_file() {
-  export USER_ID=$1
-  export GROUP_ID=$2
-  envsubst < /opt/openshift/passwd.template > /opt/openshift/passwd
-  export LD_PRELOAD=libnss_wrapper.so
-  export NSS_WRAPPER_PASSWD=/opt/openshift/passwd
-  export NSS_WRAPPER_GROUP=/etc/group
-}
+for name in $(get_is_names); do
```

Comment:
I'm ok with that.  It fits other patterns we'd have (like Drupal, which
wants to add modules from a git repo).

On Fri, Oct 9, 2015 at 9:55 AM, Michal Fojtik <notifications@github.com>
wrote:

> In 1/contrib/jenkins/run-jenkins
> <https://github.com/openshift/jenkins/pull/36#discussion_r41631987>:
>
> >
> > -# Generate passwd file based on current uid
> > -function generate_passwd_file() {
> > -  export USER_ID=$1
> > -  export GROUP_ID=$2
> > -  envsubst < /opt/openshift/passwd.template > /opt/openshift/passwd
> > -  export LD_PRELOAD=libnss_wrapper.so
> > -  export NSS_WRAPPER_PASSWD=/opt/openshift/passwd
> > -  export NSS_WRAPPER_GROUP=/etc/group
> > -}
> > +for name in $(get_is_names); do
>
> fixed (added has_service_account function to detect openshift)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/jenkins/pull/36/files#r41631987>.
>